src/cloudevents/cloudeventprotobufserializer.rs

```diff
@@ -11,23 +11,27 @@
  * SPDX-License-Identifier: Apache-2.0
  ********************************************************************************/
 
-use cloudevents::Event as CloudEvent;
 use protobuf::Message;
 
-use crate::cloudevent::serializer::{CloudEventSerializer, SerializationError};
-use crate::cloudevents::CloudEvent as CloudEventProto;
+// cloudevents-sdk
+use cloudevents::Event as CloudEvent;
+
+// protoc-generated code from cloudevents.proto
+use crate::proto_cloudevents::cloudevents::CloudEvent as ProtoCloudEvent;
+
+use crate::cloudevents::{CloudEventSerializer, SerializationError};
 
 /// Serialize and deserialize `CloudEvents` to protobuf format.
-pub struct CloudEventProtobufSerializer;
-impl CloudEventSerializer for CloudEventProtobufSerializer {
+pub struct ProtoCloudEventbufSerializer;
+impl CloudEventSerializer for ProtoCloudEventbufSerializer {
```

Comment:
this seems to be a search/replace error ...

Reply:
True